pkg/cli/root.go

```diff
@@ -68,21 +68,8 @@ and <PROJECT VERSION> a supported project version for these plugins.
 
 For more specific help for the init command of a certain plugins and project version
 configuration please run:
-    %[1]s init --help --plugins=<PLUGIN KEYS> [--project-version=<PROJECT VERSION>]
-`,
+    %[1]s init --help --plugins=<PLUGIN KEYS> [--project-version=<PROJECT VERSION>]`,
 		c.commandName, c.getPluginTable())
-
-	if len(c.defaultPlugins) != 0 {
-		if defaultPlugins, found := c.defaultPlugins[c.defaultProjectVersion]; found {
-			str += fmt.Sprintf("\nDefault plugin keys: %q\n", strings.Join(defaultPlugins, ","))
-		}
-	}
-
-	if c.defaultProjectVersion.Validate() == nil {
-		str += fmt.Sprintf("Default project version: %q\n", c.defaultProjectVersion)
-	}
```

Comment:
Why would we remove it?

Reply:
This info is already present in the description of the flag
```go
--project-version string   project version (default "3")
```
the second parameter here is the default value of the flag
https://github.com/kubernetes-sigs/kubebuilder/blob/cbc6e383c342f1337ab37ee4aa0755957a01f9c7/pkg/cli/root.go#L49-L50
which is used to generate the flag description